tests/test_eval_detection.py

```diff
@@ -185,14 +187,89 @@ def test_wrap_angle() -> None:
     assert wrap_angle(theta) == expected_result
 
 
-# TODO Stub
 def test_accumulate() -> None:
-    pass
+    """Verify that the accumulate function matches known output for a self-comparison."""
+    cfg = DetectionCfg()
+    # compare a set of labels to itself
+    cls_to_accum, cls_to_ninst = accumulate(
+        TEST_DATA_LOC / "detections",
+        TEST_DATA_LOC / "detections/1/per_sweep_annotations_amodal/tracked_object_labels_0.json",
+        cfg,
+    )
+    # ensure the detections match at all thresholds, have 0 TP errors, and have AP = 1
+    assert (
+        cls_to_accum["VEHICLE"]
+        == np.array([[1.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0, 1.0], [1.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0, 1.0]])
```

Comment:
Small nitpick -- might be a bit easier to parse if we make the values here `expected_ATE`, `expected_ASE` instead of 0, 1 etc, which i have a bit of a hard time following